### PR TITLE
Fluentbit logging doc update

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

File: docs/modules/servers/pages/distributed/operate/logging.adoc [hunk 1]
```diff
@@ -292,4 +285,54 @@ FluentBit Parser config:
...
+                        </jsonFormatter>
+                    </layout>
+                </encoder>
+        </appender>
+
+        <root level="INFO">
+                <appender-ref ref="CONSOLE" />
+        </root>
+
+        <logger name="es-error-logger" level="DEBUG" additivity="false">
```
Comment (chibenwa) [inline docs/modules/servers/pages/distributed/operate/logging.adoc:325]: `es-error-logger` is not needed with fluentBit please remove it.

**Comment by chibenwa** [comment]
`td-agent-bit.conf` example?

**Comment by Arsnael** [comment]
> `td-agent-bit.conf` example?

As stated in the doc, the fluentbit for K8s is configured to scrap logs on running containers. I looked at what we use on our preprod env, we have nothing specific for james. More like stuff to parse out things we don't need from Nodes. So no conf examples for fluentbit, the default one is enough.

**Review by chibenwa — APPROVED** [review]

**Comment by Arsnael** [comment]
test this please

**Comment by chibenwa** [comment]
Tests on a doc ? o.O

**Comment by Arsnael** [comment]
Was just for testing, killed it now

File: docs/modules/servers/pages/distributed/operate/logging.adoc [hunk 2]
```diff
@@ -292,4 +285,50 @@ FluentBit Parser config:
...
+        </appender>
+
+        <root level="INFO">
+                <appender-ref ref="CONSOLE" />
+        </root>
+
+</configuration>
+....
+
+Regarding FluentBit on Kubernetes, you need to install it as a DeamonSet. Some official template exist
```
Comment (LanKhuat) [inline docs/modules/servers/pages/distributed/operate/logging.adoc:328]: s/Deamon/Daemon

**Review by LanKhuat — APPROVED** [review]

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**